config/defines.inc.php

```diff
@@ -94,10 +94,10 @@
 
 define('_PS_TOOL_DIR_', _PS_CORE_DIR_.'/tools/');
 if (!defined('_PS_GEOIP_DIR_')) {
-    define('_PS_GEOIP_DIR_', _PS_TOOL_DIR_.'geoip/');
+    define('_PS_GEOIP_DIR_', _PS_CORE_DIR_.'/localization/');
```

Comment:
Can you move it to `app/Resources/geoip`?
This folder is only used for former XMLs, we're trying to move everything to `app/Resources` and these files will be migrated to CLDR data using standards locale (at some point :D)